app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardResultsStep/PlanWizardResultsStepActions.js

```diff
@@ -1,19 +1,72 @@
 import API from '../../../../../../../../common/API';
-import { POST_V2V_MIGRATION_PLANS } from './PlanWizardResultsStepConstants';
-import { requestMigrationPlansData } from './planWizardResultsStep.fixtures';
+import {
+  POST_V2V_MIGRATION_PLANS,
+  POST_V2V_MIGRATION_REQUESTS
+} from './PlanWizardResultsStepConstants';
+import {
+  requestMigrationPlansData,
+  requestMigrationRequestsData
+} from './planWizardResultsStep.fixtures';
 
-const _postMigrationPlansActionCreator = (url, migrationPlans) => dispatch =>
-  dispatch({
-    type: POST_V2V_MIGRATION_PLANS,
-    payload: API.post(url, migrationPlans)
-  }).catch(error => {
-    // to enable UI development without the backend ready, i'm catching the error
-    // and passing some mock data thru the FULFILLED action after the REJECTED action is finished.
+const mockMode = false;
+
+const _postMigrationPlansActionCreator = (url, migrationPlans) => dispatch => {
+  if (mockMode) {
```

Comment:
in mockMode should we bother calling the API or just go straight to dispatching the mock data?

Reply:
I have been mostly following the existing pattern in other Action creators - although I agree, we could skip calling the API.

Reply:
That's fine. I'd like to see an overall mock flag as well to be able to turn them all on from one place at some point. We can work that all out later.